src/pages/cluster/ClusterGroupForm.tsx

```diff
@@ -84,9 +85,7 @@ const ClusterGroupForm: FC<Props> = ({ group }) => {
           const verb = group ? "saved" : "created";
           navigate(
             `/ui/cluster/groups/detail/${values.name}`,
-            notify.queue(
-              notify.success(`Cluster group ${values.name} ${verb}.`),
-            ),
+            notify.queue(success(`Cluster group ${values.name} ${verb}.`)),
```

Comment:
I see, previously notify.success was setting the state so notify.queue actually wasn't working properly. Nice spot.